src/libcore/option.rs

```diff
@@ -10,9 +10,9 @@
 
 //! Optional values
 //!
-//! Type `Option` represents an optional value: every `Option`
+//! Type `Option<T>` represents an optional value: every `Option<T>`
```

Comment:
I _thought_ I included this in https://github.com/rust-lang/rfcs/blob/master/text/0505-api-comment-conventions.md, but apparently did not.

We've been preferring the type params always in the single or double param case, and doing it on a case-by-case basis with more than three. `Option<T>` should use it.